it/sender_util.go

```diff
@@ -2,246 +2,106 @@ package it
 
 import (
 	"bufio"
-	"crypto/tls"
 	"encoding/json"
 	"fmt"
-	"io"
 	"log"
-	"net/http"
-	"net/http/cookiejar"
 	"net/url"
 	"os"
-	"path/filepath"
 	"strings"
-	"time"
 )
 
-func SendBeacons(oldStylePath string, newStylePath string) {
-
-	cookieJar, _ := cookiejar.New(nil)
-	tr := &http.Transport{
-		TLSClientConfig:    &tls.Config{InsecureSkipVerify: true},
-		MaxIdleConns:       100,
-		IdleConnTimeout:    10 * time.Second,
-		DisableCompression: true,
-	}
-
-	client := &http.Client{Transport: tr,
-		Jar: cookieJar,
-		CheckRedirect: func(req *http.Request, via []*http.Request) error {
-			return http.ErrUseLastResponse
-		}}
-
-	if oldStylePath != "" {
-		valuesSrcOld := getRealBeaconsOldStyle(oldStylePath)
-
-		for i, v := range valuesSrcOld {
-			httpPostFormOldStyle(v, client, i)
-		}
-	}
-
-	if newStylePath != "" {
-		valuesSrcNew := getRealBeaconsNewStyle(newStylePath)
-
-		for i, v := range valuesSrcNew {
-			httpPostFormNewStyle(v, client, i)
-		}
-	}
-}
-
-func httpPostFormOldStyle(params url.Values, client *http.Client, cnt int) {
-	uaStr := params.Get("user.agent")
-
-	if len(uaStr) == 0 {
-		uaStr = "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36"
+func processFile(file string) ([]url.Values, error) {
```

Comment:
processFile is used by oldStyleBeaconSender. Move it to file old_style_beacon_sender.go

Reply:
make if part of structure oldStyleBeaconSender